README.md

```diff
@@ -323,6 +323,10 @@ as `onNext`, `onComplete`, or `cancel`.
 
 `take%%<n>%%<id>` : The driver should hang the test thread until it has received at least `n` items IN TOTAL, and then immediately cancel.
 
+`disconnect` : The driver should disconnect the client from the server.
```

Comment:
This is just to be used by the client. 

tck-server run on DSL so this will only be used by the client